### PR TITLE
add basic header component based on spec

### PR DESCRIPTION
**Pull request by aholtzman**
## Related Issue

resolves issue #14 
## Summary of Changes

1. [x] Component 
1. [x] Storybook doc
1. [x] Spec (basic)

### CODE REVIEW

File: src/components/header/header.spec.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    document.body.appendChild(header);
+
+    await header.updateComplete;
+  });
+
+  describe('Default Behavior', () => {
+    it('should not be null', () => {
+      expect(header).not.equal(undefined);
+      expect(header.querySelectorAll('header').length).equal(1);
```
Comment (thescientist13) [inline src/components/header/header.spec.js:18]: Maybe a quick test here to see if at least one `` exists and it's `src` and `alt` attributes are not undefined?

File: src/components/header/header.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    this.innerHTML = `
+      <header>
+        <a
+          class="flex justify-center p-2"
+          href="/"
+          title="Tuesdays Tunes Home page"
+        >
+          <img
+            class="w-full max-w-xs"
+            src="/assets/tuesdays-tunes.webp"
```
[the remark this answers is not in the document]
Comment (aholtzman) [reply src/components/header/header.js:12]: I have the PSD, it is pretty small, but something we can use to get started.  I will swap the file later.

File: src/components/header/header.spec.js (hunk 1 above)
Comment (aholtzman) [reply src/components/header/header.spec.js:18]: sure. I am not familiar with chai and was trying to get at least the minimum testing in place to be expanded on.

File: src/components/header/header.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (thescientist13) [reply src/components/header/header.js:12]: Ah, ok good to know.  I can ask for some revisions from the designer if needed.  I think I have a few other asset needs (avatar images) from other tasks so I can make a batch request to her as well, if that helps.

[the remark this answers is not in the document]
Comment (thescientist13) [reply src/components/header/header.js:12]: Yeah, no that you have updated I see what you mean about the logo.  If you notice any other things let me know, I'll reach out to the designer tonight with a batch of requests.  I'm tracking them over in #44

**Review by thescientist13 — APPROVED** [review]

**Pull request merged**